[EuiIcon] Add `crossInCircle` glyph

ek-so opened the pull request:
## Summary

We need an outlined icon variant for cross (agreed with @MichaelMarcialis [here](https://elastic.slack.com/archives/CKNPF3LTX/p1721910808910439))

## QA

### General checklist

- Browser QA
    - [x] Checked in both **light and dark** modes
- Designer checklist
  - [x] [Figma branch will be merged](https://www.figma.com/design/sJ01e9yxr9giTb1Inw2uOj/Cross-in-circle-icon-(outlined)?node-id=44866-64&t=OO3hn4GjDTHJH7ld-4) after the PR is approved.

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429)
* Commit: 1e64b99e8d6e99a688198a51b6f334821cb3ddcb

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429","number":2429,"commit":"1e64b99e8d6e99a688198a51b6f334821cb3ddcb"}],"number":2429}
buildkite-pr-comment-->

tkajtoch commented:
Hi @ek-so! Could you please add and commit a changelog item for this PR? You can create it by running `yarn yo-changelog 7924` in `packages/eui` directory. Let me know if you encounter any issues or need assistance!

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435)
* Commit: 181021a2ce87dfbca923ffad22c6222a0dfb1f96

### History
* :green_heart: [Build #2429](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429) succeeded 1e64b99e8d6e99a688198a51b6f334821cb3ddcb

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435","number":2435,"commit":"181021a2ce87dfbca923ffad22c6222a0dfb1f96"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429","number":2429,"commit":"1e64b99e8d6e99a688198a51b6f334821cb3ddcb"}],"number":2435}
buildkite-pr-comment-->

cee-chen commented on packages/eui/changelogs/upcoming/7924.md:
```diff
@@ -0,0 +1,2 @@
+- Added/Updated ...
```
```suggestion
- Updated `EuiIcon` with a new `crossInCircle` glyph
```

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439)
* Commit: 0475045df538180fec236598937a73ac80ee05a1

### History
* :green_heart: [Build #2438](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438) succeeded d5db90a0bc878aeda21121b41ff5512b5d6d7770
* :green_heart: [Build #2435](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435) succeeded 181021a2ce87dfbca923ffad22c6222a0dfb1f96
* :green_heart: [Build #2429](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429) succeeded 1e64b99e8d6e99a688198a51b6f334821cb3ddcb

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439","number":2439,"commit":"0475045df538180fec236598937a73ac80ee05a1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438","number":2438,"commit":"d5db90a0bc878aeda21121b41ff5512b5d6d7770"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435","number":2435,"commit":"181021a2ce87dfbca923ffad22c6222a0dfb1f96"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429","number":2429,"commit":"1e64b99e8d6e99a688198a51b6f334821cb3ddcb"}],"number":2439}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440)
* Commit: 18eb78ddc364bae255e09a6cdbea2aeb4713e43a

### History
* :green_heart: [Build #2439](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439) succeeded 0475045df538180fec236598937a73ac80ee05a1
* :green_heart: [Build #2438](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438) succeeded d5db90a0bc878aeda21121b41ff5512b5d6d7770
* :green_heart: [Build #2435](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435) succeeded 181021a2ce87dfbca923ffad22c6222a0dfb1f96
* :green_heart: [Build #2429](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429) succeeded 1e64b99e8d6e99a688198a51b6f334821cb3ddcb

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440","number":2440,"commit":"18eb78ddc364bae255e09a6cdbea2aeb4713e43a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439","number":2439,"commit":"0475045df538180fec236598937a73ac80ee05a1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438","number":2438,"commit":"d5db90a0bc878aeda21121b41ff5512b5d6d7770"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435","number":2435,"commit":"181021a2ce87dfbca923ffad22c6222a0dfb1f96"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429","number":2429,"commit":"1e64b99e8d6e99a688198a51b6f334821cb3ddcb"}],"number":2440}
buildkite-pr-comment-->

MichaelMarcialis commented on packages/eui/src/components/icon/svgs/crossInCircle.svg:
It looks as though the circle's stroke is slightly less than 1px. Any chance we could make it exactly 1px to avoid any subpixel blurriness?

![CleanShot 2024-07-29 at 15 47 31](https://github.com/user-attachments/assets/02e7da8d-1018-4873-b0c8-24b2c3797a31)

ek-so replied:
Thank Michael, sorry for all that discrepancies! Honestly, I just created my form based on outlined plus we already have. Considering other outlined icons we have, I'm a bit lost which template to stick to. You can see, these +/- differ from what's on your screenshot. It means we'll have to change all those outlined icons later too, right @MichaelMarcialis? 
![CleanShot 2024-07-30 at 10 11 17@2x](https://github.com/user-attachments/assets/2b66d19d-b620-4053-b61b-76e257071f31)

Also, the cross inside looks like also not 1px folk. Is in intentionally more for filled icon? Isn't it blurry then? Maybe we can also correct this glyph, while we're here?
![CleanShot 2024-07-30 at 10 04 55@2x](https://github.com/user-attachments/assets/6d742ddc-2d1e-4acd-92b6-88196903725c)

MichaelMarcialis replied:
Yeah, all good points. These are just some of the many discrepancies and inconsistencies we need to correct in general in the icon library. 

Anyway, for the purposes of this PR, I'd suggest proceeding with the comment recommendations, even they create inconsistency with some of those similar examples you shared. We'll eventually update those other ones to match. Alternatively, if you have the time and willingness, please do feel free to adjust them as part of this PR. But no pressure. Assuming we continue to maintain our custom icon library, I have it on my to-do list to make these sorts of adjustments in the near future.

MichaelMarcialis commented on packages/eui/src/components/icon/svgs/crossInCircle.svg:
I noticed that the endpoints for the `x` symbol don't sit perfectly on the pixel grid, unlike its sister `crossInCircleFilled` icon. Would you mind adjusting so that the endpoints are the same as `crossInCircleFilled`?

![CleanShot 2024-07-29 at 15 45 57](https://github.com/user-attachments/assets/ae5d40ec-522a-41e3-b7c8-3b32e87a7483)

MichaelMarcialis commented on packages/eui/src/components/icon/svgs/crossInCircle.svg:
We typically try to minify and strip any non-essential information from the SVG files. In this case, we could remove all `fill` attributes, the `g` container, and the `defs`. Doing so would yield SVG code similar to this:

```
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
  <path d="m8.755 8 2.64 2.641a.534.534 0 1 1-.754.755L8 8.755l-2.641 2.64a.533.533 0 1 1-.755-.754L7.244 8l-2.64-2.641a.534.534 0 1 1 .755-.755l2.64 2.641 2.642-2.64a.534.534 0 1 1 .755.754L8.755 8Zm4.904-5.66c3.12 3.121 3.12 8.199 0 11.32-3.12 3.12-8.198 3.12-11.318 0C-.78 10.538-.78 5.46 2.34 2.34c3.12-3.12 8.198-3.12 11.319 0Zm-.665.666a7.062 7.062 0 1 0-9.988 9.987 7.062 7.062 0 0 0 9.988-9.987Z"/>
</svg>

```

After making updates based on previous comments, mind scrubbing these extraneous elements from your updated SVG file?

MichaelMarcialis commented on packages/eui/src/components/icon/svgs/crossInCircle.svg:
Can we change this file's name from `crossInCircle.svg` to `cross_in_circle.svg`? I know there's currently a mix of snake and camel case in the SVGs folder currently, but we'll be updating them all to snake case going forward.

MichaelMarcialis commented on packages/eui/src/components/icon/icon_map.ts:
```diff
@@ -79,6 +79,7 @@ export const typeToPathMap = {
   createSingleMetricJob: 'ml_create_single_metric_job',
   cross: 'cross',
   crossClusterReplicationApp: 'app_cross_cluster_replication',
+  crossInCircle: 'crossInCircle',
```
Once the SVG file name is update, you'll also need to update this.

```suggestion
  crossInCircle: 'cross_in_circle',
```

MichaelMarcialis reviewed (commented):
Thanks for the PR, @ek-so! I left you some small comments for your initial review. Let me know if you have any questions.

ek-so commented:
Hey @MichaelMarcialis, thanks for your comments! To summarize:
1. I changed the border and removed unnecessary attributes from svg (`fill-rule="evenodd”` can't be deleted, icon becomes a plain circle instead of what it should be)
2. Changed the name to `cross_in_circle`
3. I tried to find filled cross among existing EUI icons to fix it as well — it seems that it doesn't exist currently in the code, only in Figma (there I fixed a shape just in case, for the future). Let me know if we want to add this icon to our set as well! Also, I'd like to help with other encircled icons, but I would better do it in a separate PR later.

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471)
* Commit: 6e514055d11d34e7b608f3f907059e47fc5411bd

### Failed CI Steps
* [:hammer: EUI pull request test](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471#01910844-f2d2-43b8-a67e-db0716eefb47)

### History
* :green_heart: [Build #2440](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440) succeeded 18eb78ddc364bae255e09a6cdbea2aeb4713e43a
* :green_heart: [Build #2439](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439) succeeded 0475045df538180fec236598937a73ac80ee05a1
* :green_heart: [Build #2438](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438) succeeded d5db90a0bc878aeda21121b41ff5512b5d6d7770
* :green_heart: [Build #2435](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435) succeeded 181021a2ce87dfbca923ffad22c6222a0dfb1f96
* :green_heart: [Build #2429](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429) succeeded 1e64b99e8d6e99a688198a51b6f334821cb3ddcb

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471","number":2471,"commit":"6e514055d11d34e7b608f3f907059e47fc5411bd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440","number":2440,"commit":"18eb78ddc364bae255e09a6cdbea2aeb4713e43a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439","number":2439,"commit":"0475045df538180fec236598937a73ac80ee05a1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438","number":2438,"commit":"d5db90a0bc878aeda21121b41ff5512b5d6d7770"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435","number":2435,"commit":"181021a2ce87dfbca923ffad22c6222a0dfb1f96"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429","number":2429,"commit":"1e64b99e8d6e99a688198a51b6f334821cb3ddcb"}],"number":2471}
buildkite-pr-comment-->

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474)
* Commit: 2d4a21412405b32bc932f9de9dc7e2adf346c3df

### Failed CI Steps
* [:hammer: EUI pull request test](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474#019108a3-c7fd-47cf-b44f-7bfd3724c50d)

### History
* :broken_heart: [Build #2471](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471) failed 6e514055d11d34e7b608f3f907059e47fc5411bd
* :green_heart: [Build #2440](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440) succeeded 18eb78ddc364bae255e09a6cdbea2aeb4713e43a
* :green_heart: [Build #2439](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439) succeeded 0475045df538180fec236598937a73ac80ee05a1
* :green_heart: [Build #2438](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438) succeeded d5db90a0bc878aeda21121b41ff5512b5d6d7770
* :green_heart: [Build #2435](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435) succeeded 181021a2ce87dfbca923ffad22c6222a0dfb1f96
* :green_heart: [Build #2429](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2429) succeeded 1e64b99e8d6e99a688198a51b6f334821cb3ddcb

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474","number":2474,"commit":"2d4a21412405b32bc932f9de9dc7e2adf346c3df"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471","number":2471,"commit":"6e514055d11d34e7b608f3f907059e47fc5411bd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440","number":2440,"commit":"18eb78ddc364bae255e09a6cdbea2aeb4713e43a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439","number":2439,"commit":"0475045df538180fec236598937a73ac80ee05a1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438","number":2438,"commit":"d5db90a0bc878aeda21121b41ff5512b5d6d7770"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435","number":2435,"commit":"181021a2ce87dfbca923ffad22c6222a0dfb1f96"}],"number":2474}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2483)
* Commit: b4185f61b7c14f9eeba50dd85da54482c6209928

### Failed CI Steps
* [:hammer: EUI pull request test](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2483#01910a20-3dd3-426b-9020-cf89b4b18630)

### History
* :broken_heart: [Build #2474](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474) failed 2d4a21412405b32bc932f9de9dc7e2adf346c3df
* :broken_heart: [Build #2471](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471) failed 6e514055d11d34e7b608f3f907059e47fc5411bd
* :green_heart: [Build #2440](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440) succeeded 18eb78ddc364bae255e09a6cdbea2aeb4713e43a
* :green_heart: [Build #2439](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439) succeeded 0475045df538180fec236598937a73ac80ee05a1
* :green_heart: [Build #2438](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438) succeeded d5db90a0bc878aeda21121b41ff5512b5d6d7770
* :green_heart: [Build #2435](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2435) succeeded 181021a2ce87dfbca923ffad22c6222a0dfb1f96

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2483","number":2483,"commit":"b4185f61b7c14f9eeba50dd85da54482c6209928"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474","number":2474,"commit":"2d4a21412405b32bc932f9de9dc7e2adf346c3df"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471","number":2471,"commit":"6e514055d11d34e7b608f3f907059e47fc5411bd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440","number":2440,"commit":"18eb78ddc364bae255e09a6cdbea2aeb4713e43a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439","number":2439,"commit":"0475045df538180fec236598937a73ac80ee05a1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2438","number":2438,"commit":"d5db90a0bc878aeda21121b41ff5512b5d6d7770"}],"number":2483}
buildkite-pr-comment-->

cee-chen commented:
@MichaelMarcialis is this PR good to go from a designer point of view?

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2500)
* Commit: 986c23379e42e69a3220dde3051ea2d11cfa6023

### History
* :broken_heart: [Build #2483](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2483) failed b4185f61b7c14f9eeba50dd85da54482c6209928
* :broken_heart: [Build #2474](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474) failed 2d4a21412405b32bc932f9de9dc7e2adf346c3df
* :broken_heart: [Build #2471](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471) failed 6e514055d11d34e7b608f3f907059e47fc5411bd
* :green_heart: [Build #2440](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440) succeeded 18eb78ddc364bae255e09a6cdbea2aeb4713e43a
* :green_heart: [Build #2439](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439) succeeded 0475045df538180fec236598937a73ac80ee05a1

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2500","number":2500,"commit":"986c23379e42e69a3220dde3051ea2d11cfa6023"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2483","number":2483,"commit":"b4185f61b7c14f9eeba50dd85da54482c6209928"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474","number":2474,"commit":"2d4a21412405b32bc932f9de9dc7e2adf346c3df"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471","number":2471,"commit":"6e514055d11d34e7b608f3f907059e47fc5411bd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440","number":2440,"commit":"18eb78ddc364bae255e09a6cdbea2aeb4713e43a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439","number":2439,"commit":"0475045df538180fec236598937a73ac80ee05a1"}],"number":2500}
buildkite-pr-comment-->

ek-so commented:
> @MichaelMarcialis is this PR good to go from a designer point of view?

@MichaelMarcialis hey hey, is it all fine by you?

MichaelMarcialis commented on packages/eui/src/components/icon/svgs/cross_in_circle.svg:
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="none" viewBox="0 0 16 16">
```
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

MichaelMarcialis commented on packages/eui/src/components/icon/svgs/cross_in_circle.svg:
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="none" viewBox="0 0 16 16">
+  <g clip-path="url(#a)">
+    <path d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708Z"/>
+    <path fill-rule="evenodd" d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0Zm-1 0A7 7 0 1 1 1 8a7 7 0 0 1 14 0Z"/>
+  </g>
```
```suggestion
  <path d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708Z"/>
  <path fill-rule="evenodd" d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0Zm-1 0A7 7 0 1 1 1 8a7 7 0 0 1 14 0Z"/>
```

MichaelMarcialis reviewed (approved):
Thanks for making those updates, @ek-so! Looks great. I left two small comments for your review, but nothing worth holding you up over. Assuming you can address those two suggestions to remove the `fill` attribute and `g` element, I'll go ahead and approve now.

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2514)
* Commit: 887e89e8c6e80851508096748fca9f3e19328c7c

### History
* :green_heart: [Build #2500](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2500) succeeded 986c23379e42e69a3220dde3051ea2d11cfa6023
* :broken_heart: [Build #2483](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2483) failed b4185f61b7c14f9eeba50dd85da54482c6209928
* :broken_heart: [Build #2474](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474) failed 2d4a21412405b32bc932f9de9dc7e2adf346c3df
* :broken_heart: [Build #2471](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471) failed 6e514055d11d34e7b608f3f907059e47fc5411bd
* :green_heart: [Build #2440](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440) succeeded 18eb78ddc364bae255e09a6cdbea2aeb4713e43a
* :green_heart: [Build #2439](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2439) succeeded 0475045df538180fec236598937a73ac80ee05a1

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2514","number":2514,"commit":"887e89e8c6e80851508096748fca9f3e19328c7c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2500","number":2500,"commit":"986c23379e42e69a3220dde3051ea2d11cfa6023"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2483","number":2483,"commit":"b4185f61b7c14f9eeba50dd85da54482c6209928"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2474","number":2474,"commit":"2d4a21412405b32bc932f9de9dc7e2adf346c3df"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2471","number":2471,"commit":"6e514055d11d34e7b608f3f907059e47fc5411bd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/2440","number":2440,"commit":"18eb78ddc364bae255e09a6cdbea2aeb4713e43a"}],"number":2514}
buildkite-pr-comment-->
